### PR TITLE
Add enroling with the DCS CA to the list of steps to integrate with DCS

### PR DESCRIPTION
**Pull request by benjamin-mortimer**
We added a guide to the pilot docs about how to enrol with the DCS certificate authority. But we didn't add enrolling with the CA to the list of steps participants take to integrate with DCS. It's an important step and comes with a lead time, so I've added it.

### CODE REVIEW

File: source/integrating-with-the-dcs.html.md.erb [hunk 1]
```diff
@@ -24,7 +24,8 @@ Your client will need to be able to sign and encrypt JSON data and make HTTP req
 
 To build a working DCS client, you'll need to:
 
-1. [Generate keys and request certificates](/generate-keys-and-request-certificates/#generate-keys-and-request-certificates) - you’ll [contact DCS support](/support/#support) to get the Public Key Infrastructure (PKI) information so you can get started with integrating with the DCS.
+1. [Enrol with the DCS certificate authority](https://dcs-pilot-docs.cloudapps.digital/enrol-with-the-DCS-certificate-authority/#enrol-with-the-dcs-certificate-authority)
```
Comment (PippaClarkGDS) [inline source/integrating-with-the-dcs.html.md.erb:27]:
```suggestion
1. [Enrol with the DCS certificate authority](https://dcs-pilot-docs.cloudapps.digital/enrol-with-the-DCS-certificate-authority/#enrol-with-the-dcs-certificate-authority).
```

**Review by PippaClarkGDS — COMMENTED** [review]
tiny change here

**Comment by benjamin-mortimer** [comment]
Thanks @PippaClarkGDS - I've committed your suggestion

File: source/integrating-with-the-dcs.html.md.erb (hunk 1 above)
Comment (PippaClarkGDS) [inline source/integrating-with-the-dcs.html.md.erb]:
```suggestion
1. [Enrol with the DCS certificate authority](https://dcs-pilot-docs.cloudapps.digital/enrol-with-the-DCS-certificate-authority/#enrol-with-the-dcs-certificate-authority).
```

**Review by PippaClarkGDS — APPROVED** [review]

**Pull request merged**